File: keycloak-client/src/main/java/cc/coopersoft/accesstoken/keycloak/model/KeycloakCodeResult.java
package cc.coopersoft.accesstoken.keycloak.model;

import cc.coopersoft.accesstoken.api.callback.CodeRequestCallback;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
public final class KeycloakCodeResult implements CodeRequestCallback.Result {
  private long expiresIn;
  private Date created;
}

File: keycloak-client/src/main/java/cc/coopersoft/accesstoken/keycloak/model/KeycloakCodeResult.kt
package cc.coopersoft.accesstoken.keycloak.model

import cc.coopersoft.accesstoken.api.callback.CodeRequestCallback
import java.util.Date

data class KeycloakCodeResult(override val expiresIn: Long, override var created: Date?) :
    CodeRequestCallback.Result {
    constructor() : this(0, null)
}